Jellyfin.Api/Auth/FirstTimeSetupPolicy/FirstTimeSetupHandler.cs

```diff
@@ -40,6 +40,13 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
             }
 
             var contextUser = context.User;
+            if (contextUser.GetIsApiKey())
```

Comment:
Shouldn't this just check for Admin role? We have too much authz logic tied to API Keys tbh. API keys and real admin users should both have the Administrator role.